From stable-bounces@linux.kernel.org  Fri Feb  3 12:22:26 2006
Date: Fri, 3 Feb 2006 21:15:34 +0100
From: Adrian Bunk <bunk@stusta.de>
To: stable@kernel.org
Cc: linville@tuxdriver.com
Subject: [PATCH] PCMCIA=m, HOSTAP_CS=y is not a legal configuration

CONFIG_PCMCIA=m, CONFIG_HOSTAP_CS=y doesn't compile.

Reported by "Gabriel C." <crazy@pimpmylinux.org>.

This patch was already included in 2.6.16-rc2.

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/net/wireless/hostap/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.15.3/drivers/net/wireless/hostap/Kconfig
===================================================================
--- linux-2.6.15.3.orig/drivers/net/wireless/hostap/Kconfig
+++ linux-2.6.15.3/drivers/net/wireless/hostap/Kconfig
@@ -61,7 +61,7 @@ config HOSTAP_PCI
 
 config HOSTAP_CS
 	tristate "Host AP driver for Prism2/2.5/3 PC Cards"
-	depends on PCMCIA!=n && HOSTAP
+	depends on PCMCIA && HOSTAP
 	---help---
 	Host AP driver's version for Prism2/2.5/3 PC Cards.
 
